packages/babel-helpers/src/helpers.js

```diff
@@ -585,3 +599,18 @@ helpers.toConsumableArray = template(`
     }
   });
 `);
+
+helpers.toPropertyKey = template(`
+  (function (key) {
+    if (typeof key === "symbol" || (
+      typeof key === "object" && 
```

Comment:
I think it's due to the linter's 110 column length requirement. I'm thinking of alternatives but I don't know if they're a significant improvement. How about this:

```
var isPolyFilledSymbol = typeof key === "object" && 
      key !== null &&
      typeof key.toString === "function" && 
      key.toString().slice(0, 6) === "Symbol(");
if (typeof key === "symbol" || isPolyfilledSymbol) {
```